docs/modules/ROOT/pages/guides/src5-migration.adoc

```diff
@@ -152,16 +118,13 @@ mod MigratingContract {
     (...)
 
     #[external(v0)]
-    fn migration_initializer(ref self: ContractState) {
-        // Add appropriate Access Control mechanism
+    fn migrate(ref self: ContractState) {
+        // WARNING: Missing Access Control mechanism. Make sure to add one
 
         // Invoke `initialize` so this function can only be called once
         self.initializable.initialize();
```

Comment:
I'm not very sure about this part, it might conflict with pre existing `initializable` usage. I've been thinking we could have a modified version like migrations module, which is like a nonce you use + burn. I.e. you can only run migration number `3` once. Not for this PR obviously.

I think i'd add a disclaimer about previous initializable usage or something